### PR TITLE
docs(guides): flux and capsule for multi-tenant gitops

### LINKED ISSUE #528: Document how Capsule integrates in a Flux GitOps based environment
# Describe the feature
Document how Capsule integrates in a Flux GitOps based environment.

# What would the new user story look like?
The cluster admin can learn how to configure a Flux GitOps based environment with Capsule

# Expected behaviour

A detailed user guide is provided into documentation.

### PR DESCRIPTION
**Pull request by maxgio92**
Closes #528

### CODE REVIEW

File: docs/content/guides/flux2-capsule-gitops-multitenancy.md [hunk 1]
```diff
@@ -0,0 +1,566 @@
...
+
+As GitOps finds its roots in DevOps culture and it's an evolution continuous delivery techniques. As it enables a common language, the barrier to share responsibilities has been lowered.
+
+This is the motivation that led to the importance of managing multi-tenant configurations the GitOps way, and furthermore, to provide tenants the ability to control and manage their space on their own.
+
+We'll see how to do it with Flux v2 and Capsule.
+
+## Flux
+
+Flux is a set of solutions for GitOps continuous delivery, which are part of the GitOps Toolkit operators.
```
Comment (prometherion) [inline docs/content/guides/flux2-capsule-gitops-multitenancy.md:11]: Please, add links to projects.

**Review by prometherion — CHANGES_REQUESTED** [review]
I think that rather than a guide this seems a very detailed article or blog post for its level, very well done!

For the documentation website I think we should just need to add the required info without the long story that could be then exposed on other platforms, so please, try to keep it short considering that if people would like to integrate Capsule with Flux they already know all the basic concepts.

Besides that, you have to add the documentation guide link in the `gridsome.server.js` file to get it published in the navigation section.

**Comment by bsctl** [comment]
> Moreover, I think we should provide a final PoC repository, migrating and cleaning up the one of mine [flux2-capsule-multi-tenancy](https://github.com/maxgio92/flux2-capsule-multi-tenancy).

@maxgio92 the detailed tutorial can be part of this dedicated PoC repo and leave the short one in the main Capsule documentation. Anyway, let's to link repo each other so that the reader of main Capsule documentation can access the detailed tutorial.

File: docs/content/guides/flux2-capsule-gitops-multitenancy.md (hunk 1 above)
Comment (maxgio92) [reply docs/content/guides/flux2-capsule-gitops-multitenancy.md:11]: Thank you, good catch.

**Comment by maxgio92** [comment]
Thanks @bsctl and @prometherion. I'm going to fix all the parts.

**Comment by maxgio92** [comment]
I stripped down introductions to make it a real guide. I left diagrams to help understand the setup, but they can be cleaned up too if you want to further lighen it @prometherion @bsctl.

**Review by bsctl — APPROVED** [review]
lgtm @maxgio92 astonishing!!

**Pull request merged**